Algorithms_Concepts/Factorial.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+namespace Algorithms_Concepts
+{
+    public class Factorial
+    {
+       public int number;
+        public void AcceptNumber()
+        {
+            
+            Console.WriteLine("Enter a Number");
+            number = Convert.ToInt32(Console.ReadLine());
+            
+        }
+        public void DisplayFactorial()
+        {
+            int result;
+            result = CalculateFactorial(number);
+            Console.WriteLine("Factorial = "+ result);
+        }
+        public int CalculateFactorial( int number)
+        {
+           if(number==0 || number==1 )
+           {
```

Comment:
curly braces are nor required for a single line of code in it.

Reply:
done